File: CAPSTONE-PROJECT-FRONTEND_FINAL_UPDATE-master/src/assets/Project.ts
export interface Project{
    name: string;
    members:string[];
    columns:{
        [columnName: string]: Task[];
    };
}

export interface Task {
    name: string;
    content: string;
    priority: string;
    createDate: string;
    deadline: string;
    assignee: string;
    status: string;
    members: string[];
  }

File: Capstone_Project_Final_anban_Backend-main/KanbanProject-main/kanban/src/main/java/com/kanban/kanban/domain/Project.java
package com.kanban.kanban.domain;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

import java.util.List;
import java.util.Map;

@AllArgsConstructor
@NoArgsConstructor
@Data
@Document
public class Project {
    @Id
    private String name;
    private List<String> members;
    private Map<String,List<Task>> columns;

}
